code/game/gamemodes/wizard/rightandwrong.dm

```diff
@@ -7,9 +7,7 @@
 	for(var/mob/living/carbon/human/H in player_list)
 		if(H.stat == DEAD || !(H.client))
 			continue
-		if(is_special_character(H))
```

Comment:
Do me a favor and add `if(summon_type && iswizard(H))` here, keeping the continue.

Wizards shouldn't get summoned magic stuff seeing as they're intended to spend a spellpoint on it and it has a chance of giving the staves.